.github/workflows/release.yml

```diff
@@ -119,7 +119,7 @@ jobs:
 
       - name: "GoReleaser: Release"
         if: startsWith(github.ref, 'refs/tags/')
-        uses: goreleaser/goreleaser-action@v5
+        uses: goreleaser/goreleaser-action@v6
```

Comment:
@JanMa This would've actually broken our release pipelines fwiw, since the `--debug` flag added below is no longer supported on `v6` of the action :-D